apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -84,7 +84,16 @@
 {{/parse_error}}
 {{#missing_input}}
     {{#label}}<span>{{{label}}}</span>{{/label}}
-    <button type="button" class="btn btn-sm btn-secondary small border" data-placement="auto" data-toggle="popover" data-html="true" title="Missing Input" data-content="There is no submitted value for this field.  This may have happened because the question was changed by course staff after the answer was submitted."> Why <i class="fa fa-question-circle" aria-hidden="true"></i></button>
+    <button
+        type="button"
+        class="badge text-dark badge-missing-input btn btn-sm btn-secondary small border"
+        data-placement="auto"
+        data-toggle="popover"
+        data-html="true"
+        title="Missing Input"
```

Comment:
Nit, we generally prefer sentence case.

```suggestion
        title="Missing input"
```

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/10701/commits/542af0986ce39fb5b313c7c0e20d26d925e51159